help/access/file-shares.md

```diff
@@ -1,302 +1,237 @@
-   > files, toggle **Index Files** in the **Share...** dialog).
+The **Directory** section shows the contents of your current folder with three viewing modes: icons, cards, and grid. You can click an object to view its content in the **Preview**, or right-click it to access available context actions. You can also use the search bar located above the **Directory** to search for files and folders within your current directory. The search bar allows you to search for items by name, file extension, or metadata.
 
-## Manage file shares
+The **Preview** is a context-sensitive view that adapts to the the selected object. For folders, the **Preview** generates a [treemap](../visualize/viewers/tree-map.md) that highlights the largest items. For files, the functionality varies based on the file's format and data properties. It includes custom viewers for [supported formats](supported-formats.md), such as interactive spreadsheets for displaying tabular data, cell and image renderers, and chemical and biological structure viewers. You can also view the content of ZIP files and edit Markdown, TXT, and HTML files.
 
-Subject to your privileges, you can use the **Data Explorer** panel to inspect and quickly adjust access permissions to
-file shares, send comments to those you're sharing with, and more.
+![File browsing and preview](file-manager-file-browsing.gif)
 
-1. First, select the connection, file, or folder.
-2. Then navigate to **Data Explorer** on the left and expand the **Sharing** info panel to see the complete list of
-   users with access and their privileges. From here, you can click any user or group to see their profile, your
-   conversation history with them, send them a note, and more.
-3. Use the buttons provided to share access with more users, revoke access, or edit permissions to the shared item.
+:::note
 
-   > Note: The same actions are available from the context menu.
+File preview is limited to files under 10MB. The platform won't display larger files. Unsupported file formats cannot be previewed, but you can download them.
 
-<!--TBD: GIF pending changes in the UI-->
+:::
 
-## Managing folders and files
+:::note developers
+  
+You can [add custom formats using package extensions](../develop/how-to/create-package.md). In addition, you can create organization-specific previews:
 
-The **File Manager** lets you perform standard file and folder actions such as open, download, delete, or rename. To see
-a complete list of available actions, right-click the file or folder.
+<details>
+<summary> Example: Create custom file viewers </summary>
 
-> Tip: The same list of actions is available from the **Data Explorer** on the right under the **Actions** info panel.
+In this example, a script is executed against the folder content. If the folder contains files that match the file extension parameter PDB, the **Preview** displays a custom NGL viewer to visualize the molecule.
```

Comment:
"In this example, a script is executed against the folder content. If the folder contains files that match the file extension parameter PDB, the **Preview** displays a custom NGL viewer to visualize the molecule"
=>
3D molecule viewer is used for previewing the .pdb file

Reply:
I may be wrong but again, I think you have a familiriaty bias. It's not obvious what actually happens based on just one sentence you propose.